### DISCUSSION TITLE
When to push to 0.8.0beta4?

### DISCUSSION THREAD

**Discussion by tubalainen** [blakeblackshear/frigate#515] [General]
There´s a TON of new great stuff coming and also lots of fixes.

How far away are we to push beta4?

**Comment by blakeblackshear** [comment]
I haven't even pushed half of the changes yet, but this should wrap up 0.8.0. The next release will be RC1 and it will have yet another set of breaking changes.

**Reply by tubalainen** [reply]
Ah, okay. So we skip beta4 and goes straight to RC1. :) Cool!
Yes indeed there will be breaking changes.
